generators/app/prompts.js

```diff
@@ -101,21 +113,39 @@ function entitiesChoices(generator) {
 }
 
 function askForOperations(generator) {
+    const boostrapServersUnitPattern = /^((([a-zA-Z0-9-]+)(\.?))+(:[0-9]+)?)$/;
+    function isNotValidBootstrapServerString(input) {
+        return _.isEmpty(input) || input.split(',').some(bootstrapServer => boostrapServersUnitPattern.test(bootstrapServer) === false);
+    }
+
     const prompts = [
         {
             when: !generator.options['skip-prompts'],
             type: 'confirm',
             name: 'cleanup',
             message: 'Do you want to clean up your current Kafka configuration?',
             default: false
+        },
+        {
+            when: !generator.options['skip-prompts'],
```

Comment:
Should we prompt user on the `bootstrap.servers` if he didn't choose to clean up?